lib/handle_schedule.js

```diff
@@ -86,9 +91,16 @@ async function handleSchedule() {
       });
 
       core.info(`${pullRequest.html_url} merged`);
+      mergedPRList = mergedPRList + pullRequest.html_url;
     } catch (err) {
+      // Logging error messages
       core.info(`Unable to merge ${pullRequest.html_url}`);
       core.info(`The Error is : ${err}`);
+      
+      // Setting value of pull request which unable to merge because of above error[Mostly merge conflict]
+      // Setting in MC_PR_NO which can be used in Github action
+      core.info(`::set-output name=MC_PR_NO::${pullRequest.html_url}`);
+
```

Comment:
can you update the example`merge-schedule.yml` file in readme.md on how to use the job outputs?

setting the output directly inside the action might make this repo specific to our own use case. (it might not make sense to merge to upstream repo). but let's go with this solution for now.

Reply:
Done